src/Renderer/TwigTemplateBlockRenderer.php

```diff
@@ -36,7 +36,7 @@ public function render(TemplateBlock $templateBlock, array $context = []): strin
             return $cacheValue;
         }
 
-        if (strpos(SyliusKernel::VERSION, '1.12') === 0 && interface_exists(ContextProviderInterface::class)) {
+        if (strpos(SyliusCoreBundle::VERSION, '1.12') === 0 && interface_exists(ContextProviderInterface::class)) {
```

Comment:
Hi @MarcoPistorelloAtFiftyDeg , maybe it's better to investigate a little bit more and use `VERSION_ID` in order to use `>=` comparison and be ready for next sylius version? By this way, the plugin will not be compatible with the upcoming `1.13.x` Sylius version. Thanks!

Reply:
Hi @FilippoTascaAtFiftyDeg, I set the SyliusCoreBundle::VERSION_ID >= '112' instead of SyliusCoreBundle::VERSION_ID >= '11200', in order to leave the code as essential as we can.